src/styles/index.scss

```diff
@@ -1,3 +1,80 @@
+$font-family: Arial, sans-serif;
+$font-size: 30px;
+$day-border: 1px;
+$day-size: 100px;
+$day-color: #eee;
+$hover-color: #FFBFCB;
+$animation-duration: 0.5ms;
+$days-gap: 10px;
+
+$week-days:
+    "mon" 0,
+    "tue" 1,
+    "wed" 2,
+    "thu" 3,
+    "fri" 4,
+    "sat" 5,
+    "sun" 6;
+// $day та $marginNumber
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: box-sizing;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+display: flex;
+flex-wrap: wrap;
+margin: auto;
+width: calc(7 * #{$day-size} + 6 * #{$day-border} + 20px);
```

Comment:
The width calculation seems to be incorrect. If '$day-border' represents the border size, it should not be multiplied by 6 for the width calculation. Instead, use the '$days-gap' variable to account for the gaps between days.